packages/trading/src/types.ts

```diff
@@ -282,4 +282,10 @@ export interface PostTradeAdditionalParams {
    */
   signingScheme?: SigningScheme
   customEIP1271Signature?: (orderToSign: UnsignedOrder, signer: AbstractSigner<Provider>) => Promise<string>
+
+  /**
+   * By default, the quote adjustments are applied to the order.
```

Comment:
For me the description of this was not enough to understand. 

Could you indicate that this "quote adjustments" to applying to the sold/bought amounts the gas costs, slippage, partner fees, etc.

Setting it to `false` will post the order with the un-modified amounts indicated in the parameters. Note that in this case it is assumed that these concepts were already applied by the caller of the function.